<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="jest tests" tests="9" failures="1" errors="0" time="1.016">
  <testsuite name="Teste basico 2" errors="0" failures="0" skipped="0" timestamp="2024-05-13T04:46:54" time="0.727" tests="2">
    <testcase classname="Teste basico 2 Valores " name="Teste basico 2 Valores " time="0.01">
    </testcase>
    <testcase classname="Teste basico 2 CalcularCubo com string " name="Teste basico 2 CalcularCubo com string " time="0">
    </testcase>
  </testsuite>
  <testsuite name="Teste basico 1" errors="0" failures="0" skipped="0" timestamp="2024-05-13T04:46:54" time="0.726" tests="2">
    <testcase classname="Teste basico 1 Valores corretos" name="Teste basico 1 Valores corretos" time="0.012">
    </testcase>
    <testcase classname="Teste basico 1 CalcularCubo de 3 " name="Teste basico 1 CalcularCubo de 3 " time="0">
    </testcase>
  </testsuite>
  <testsuite name="Teste Darf" errors="0" failures="1" skipped="0" timestamp="2024-05-13T04:46:54" time="0.752" tests="5">
    <testcase classname="Teste Darf should throw an error when parameter types are wrong" name="Teste Darf should throw an error when parameter types are wrong" time="0.013">
    </testcase>
    <testcase classname="Teste Darf is the darf fee right?" name="Teste Darf is the darf fee right?" time="0">
    </testcase>
    <testcase classname="Teste Darf is the darf fee wrong?" name="Teste Darf is the darf fee wrong?" time="0">
    </testcase>
    <testcase classname="Teste Darf is the avg price right?" name="Teste Darf is the avg price right?" time="0">
    </testcase>
    <testcase classname="Teste Darf is this selling profitable?" name="Teste Darf is this selling profitable?" time="0">
      <failure>TypeError: darf.getProfitOrLost is not a function
    at Object.getProfitOrLost (C:\dev\projeto_pesIII\test\darf.test.js:28:21)
    at Promise.then.completed (C:\dev\projeto_pesIII\node_modules\jest-circus\build\utils.js:298:28)
    at new Promise (&lt;anonymous&gt;)
    at callAsyncCircusFn (C:\dev\projeto_pesIII\node_modules\jest-circus\build\utils.js:231:10)
    at _callCircusTest (C:\dev\projeto_pesIII\node_modules\jest-circus\build\run.js:316:40)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at _runTest (C:\dev\projeto_pesIII\node_modules\jest-circus\build\run.js:252:3)
    at _runTestsForDescribeBlock (C:\dev\projeto_pesIII\node_modules\jest-circus\build\run.js:126:9)
    at _runTestsForDescribeBlock (C:\dev\projeto_pesIII\node_modules\jest-circus\build\run.js:121:9)
    at run (C:\dev\projeto_pesIII\node_modules\jest-circus\build\run.js:71:3)
    at runAndTransformResultsToJestFormat (C:\dev\projeto_pesIII\node_modules\jest-circus\build\legacy-code-todo-rewrite\jestAdapterInit.js:122:21)
    at jestAdapter (C:\dev\projeto_pesIII\node_modules\jest-circus\build\legacy-code-todo-rewrite\jestAdapter.js:79:19)
    at runTestInternal (C:\dev\projeto_pesIII\node_modules\jest-runner\build\runTest.js:367:16)
    at runTest (C:\dev\projeto_pesIII\node_modules\jest-runner\build\runTest.js:444:34)
    at Object.worker (C:\dev\projeto_pesIII\node_modules\jest-runner\build\testWorker.js:106:12)</failure>
    </testcase>
  </testsuite>
</testsuites>